x-pack/test/common/utils/server_route_repository/create_admin_service_from_repository.ts

```diff
@@ -97,6 +98,18 @@ export async function getAdminApiClient<TServerRouteRepository extends ServerRou
   };
 }
 
+export async function getCustomRoleApiClient<TServerRouteRepository extends ServerRouteRepository>(
+  st: ReturnType<typeof RoleScopedSupertestProvider>
+): Promise<RepositorySupertestClient<TServerRouteRepository>> {
+  return await getApiClient(st, 'customRole');
```

Comment:
In #217882 I'm changing the custom role naming and you will have to get the value via `samlAuth.getCustomRole()`

Could you make a change so that it won't be broken in the follow-up PR?